packages/js-dpp/lib/blockTimeWindow/isTimeInBlockTimeWindow.js

```diff
@@ -0,0 +1,17 @@
+const getBlockTimeWindowRange = require('./getBlockTimeWindowRange');
+
+/**
+ *
+ * @param {number} lastBlockHeaderTime
+ * @param {number} timeToCheck
+ * @returns {boolean}
+ */
+function isTimeInBlockTimeWindow(lastBlockHeaderTime, timeToCheck) {
+  const {
+    timeWindowStart,
+    timeWindowEnd,
+  } = getBlockTimeWindowRange(lastBlockHeaderTime);
```

Comment:
We call this function two times. What if we rename `isTimeInBlockTimeWindow`  to `validateTimeInBlockTimeWindow` and return `ValidationResult`. In this case we can add data to validation result with time window start and end

Reply:
But we use different errors every time

Reply:
Agreed to create own `ValidationResult` class with `isValid`, `getTimeWindowStart` and `getTimeWindowEnd`.